podcasts/Onboarding/LoginCoordinator.swift

```diff
@@ -123,6 +124,10 @@ extension LoginCoordinator {
                     Analytics.track(.userSignedIn, properties: ["source": provider])
                 }
 
+                if FeatureFlag.endOfYear2024.enabled {
```

Comment:
@SergioEstevao @bjtitus few questions:
• Should we trigger the notification only for non new user?
• Do you think it's better to do it before or after the sync is completed?

Reply:
My understanding is that we have code to load the stats, so the sync should be taken in account, we also check for eligibility so if it's a new user it will probably not be eligible in terms of stats to have EOY available?